### PR TITLE
added a readme.txt

### PR DESCRIPTION
**Pull request by sidneyh**
The first time you run the app, the user has no idea what the values you require in the Add Account box. This is a doc that tells them the basics.

### CODE REVIEW

File: CSharp/BatchExplorer/readme.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿Once the app is running, you need to configure it with data from your Azure subscription.
```
Comment (matthchr) [inline CSharp/BatchExplorer/readme.txt]: Can you change this to: "Once BatchExplorer is running, you need to configure it with data about your Azure Batch account.

File: CSharp/BatchExplorer/readme.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+1. On the Accounts menu click "Add Default Account Manager". The "Add Account" dialog opens.
+2. In the Account Alias box, type in any name you like. Use the name to create distinct batch jobs.
```
Comment (matthchr) [inline CSharp/BatchExplorer/readme.txt]: Instead:
"In the Account Alias box, type in any name you like. This is the friendly name of the Batch Account and will show up in the BatchExplorer UI."

File: CSharp/BatchExplorer/readme.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+3. For the Batch Service Url, go to your subscription and copy the URL. Paste it into the box.
+4. For the Account box, go to your subscription and copy the name of the Batch. Paste it into the box.
```
Comment (matthchr) [inline CSharp/BatchExplorer/readme.txt]: For the Account box, go to your subscription and copy the name of the Batch Account. Paste it into the box.

File: CSharp/BatchExplorer/readme.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+5. For the Key box, go to your suscription and copy the Primary Key, then paste it into the box.
+
+The values are saved automatically, and loaded when you run the file again.
```
Comment (matthchr) [inline CSharp/BatchExplorer/readme.txt]: instead: "After you press ok, the values are saved and can be re-loaded whenever you run BatchExplorer"

File: CSharp/BatchExplorer/readme.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+By default, a "testbatch" is created for you using the URL, account, and key values.
```
Comment (matthchr) [inline CSharp/BatchExplorer/readme.txt]: I am not sure what this comment means - can you remove it?

**Comment by matthchr** [comment]
@sidneyh I like the idea - but made a few wording/clarification suggestions.  If you make the suggested changes and update your PR we'll accept it.

Thanks!

**Comment by sidneyh** [comment]
Alright. No problem.

**Pull request merged**